hypervisor/src/mshv/mod.rs

```diff
@@ -1449,6 +1452,7 @@ pub struct MshvVm {
     fd: Arc<VmFd>,
     msrs: Vec<MsrEntry>,
     dirty_log_slots: Arc<RwLock<HashMap<u64, MshvDirtyLogSlot>>>,
+    sev_snp_enabled: bool,
```

Comment:
It should be gated behind feature "sev_snp".

Reply:
Fixed